### PR TITLE
feat: better signup flow

### LINKED ISSUE #51: [FEATURE] <Better sign up flow>
### Description

When a user signs up at /signup, confirming the email is required. I think we should make that clear.

Also, we should probably add some info about how to find the signup page (/signup) and the signin page (/signin) either on the site itself or in README.

### PR DESCRIPTION
**Pull request by olamide203**
### 🛠️ Fixes Issue
Closes #51  

### 👨‍💻 Changes proposed
- added a `confirm-email` page which user gets redirected to on signup

### CODE REVIEW

**Review by ykdojo — APPROVED** [review]
looks good, thank you!

File: pages/confirm-email.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import { type NextPage } from 'next';
+
+const confirmEmail: NextPage = () => {
```
Comment (iShibi) [inline pages/confirm-email.tsx:5]:
```suggestion
const ConfirmEmail: NextPage = () => {
```

File: pages/confirm-email.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+					</div>
+					<div className='text-center text-gray-700'>
+						<p>Thanks for signing up! We&#39;ve sent you an email to confirm your account.</p>
+					</div>
+				</div>
+			</main>
+		</>
+	);
+};
+export default confirmEmail;
```
Comment (iShibi) [inline pages/confirm-email.tsx:24]:
```suggestion
export default ConfirmEmail;
```

**Review by iShibi — CHANGES_REQUESTED** [review]
Use `PascalCase` for page components

**Comment by iShibi** [comment]
We need to handle the redirect when a user clicks on "Confirm Email" in the mail. Also, rebase this PR with `main` to resolve the conflicts.

**Comment by olamide203** [comment]
The redirect url can be set in supabase, it defaults to the `index/` page.  after doing a  rebase there is still a conflict with `main`

**Comment by ykdojo** [comment]
@olamide203 I think you should be able to resolve it in your editor? (either local or here)

Opening a new PR is okay, too

**Comment by iShibi** [comment]
@olamide203 You didn't rebase correctly. It has been fixed now. Check the diff and let us know if you're ready to merge.

**Comment by olamide203** [comment]
sorry about that, i'm still new to the `git rebase` command and not very confident using it. everything looks good now to merge, Thanks.

**Pull request merged**